build/container/amazon-opensearch-serverless.Dockerfile

```diff
@@ -0,0 +1,3 @@
+# OpenSearch
+# See https://hub.docker.com/r/opensearchproject/opensearch/tags
```

Comment:
```suggestion
# See https://hub.docker.com/r/opensearchproject/opensearch/tags
#
# IMPORTANT! When updating the version of OpenSearch in this Dockerfile,
# make sure to update `version.org.opensearch.latest` property in a POM file,
# and to update the version in opensearch.Dockerfile as well.
```

Reply:
Note sure it's worth having two separate dockerfiles, BTW... things would be simpler if we just used the OpenSearch dockerfile for the amazon-opensearch-serverless distribution.

Reply:
yeah... I've only had this file to make the groovy script simple:
```groovy
def fromFile = parseImage( './build/container/${test.elasticsearch.distribution}.Dockerfile')
```
if we want to keep one file, then this would need to have some `if-else` logic; so maybe, let's keep the file for now, I suppose if the dependabot integration will work as we expect then we wouldn't need to update these files anyway. 🤞🏻😃